docs/docs/evaluation/index.md

```diff
@@ -22,6 +13,24 @@ The CLI evaluates the chatbot's performance by comparing the chatbot's responses
 
 The Evaluation CLI works by running a series of commands that generate, evaluate, and report on the performance of your chatbot.
 
+Here's a high-level overview of the process:
+
+1. **Generate Data**: Generate data for evaluation. You can pull data from test cases files, databases, or APIs. Generated data outputs to a `GeneratedDataStore`.
+2. **Evaluate Data**: Evaluate the generated data. You can evaluate the quality, faithfulness, and other aspects of the data. Evaluation outputs to the `EvaluationStore`.
+3. **Report on Data**: Generate reports on an evaluation run. Reports output to the `ReportStore`.
+
+```mermaid
```

Comment:
nit: consider moving this above the list. I think it's short enough that it doesn't push them below the fold + IMO it's more natural to have text below the diagram.